src/main/java/entity/UsernameValidatorService.java

```diff
@@ -0,0 +1,12 @@
+package entity;
+
+public class UsernameValidatorService implements UsernameValidator{
```

Comment:
After testing this code, the UsernameValidator seems to work as intended also, with a test done on a 1 character username being successful.